http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -695,6 +704,19 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
     }
   }
 
+  private[util] def createAzureManagedIdentity(app: App, namespacePrefix: String, workspaceId: WorkspaceId)(implicit
```

Comment:
TOL, should these managed identities now be created with `COPY_DEFINITION` cloning instructions?

Reply:
+1 (just commented too) -- I think Leo should set clone instructions when creating identities and databases.

Reply:
Based on my understanding we wanted to do this after changes were made to WSM to handle apps cloning. Is that not right? I had understood that if we create apps and identities with `COPY_DEFINITION` and WSM doesn't have cloning implemented things might break and so the plan was to make this change after WSM work.

Reply:
Ohh that makes total sense. Thanks 👍

Reply:
I guess this depends on what order things merge in (if we merge in an incomplete version of database cloning). But otherwise it shouldn't cause an issue since we only are trying to clone resources specifically enumerated in WSM, and we can just wait to include those resources in WSM until they are fully ready to be cloned (with the last piece of database cloning). The alternative is that we have another small PR later in Leo to send the new clone instructions, and all apps created between now and then would not be cloneable... open to further discussion tomorrow!